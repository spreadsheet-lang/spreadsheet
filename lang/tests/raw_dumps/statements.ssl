//@ only-host: never
D1=0

A1="apple"
A2 = "banana"
A3 = "cherry"

B1 = enum A1:A3
alias fruits = B1

type C1:C4 = $fruits
type D[:] = int
C1 = "apple"
C2 = "banana"
C3 = "apple"
C4 = "apple"
D1 = 3
D1 = 4
D1 = 2
D1 = 0
