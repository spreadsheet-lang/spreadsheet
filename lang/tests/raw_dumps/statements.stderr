found 't' at 100..101
Error: Parse
